scripts/lldb/firestore.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ast
+import json
+
+"""
+LLDB type summary providers for common Firestore types.
+
+This is primarily useful for debugging Firestore internals. It will add useful
+summaries and consolidate common types in a way that makes them easier to
+observe in the debugger.
+
+Use this by adding the following to your ~/.lldbinit file:
+
+    command script import ~/path/to/firebase-ios-sdk/scripts/lldb/firestore.py
```

Comment:
I wonder if this would be better extracted to a `README` file to be placed in `scripts/lldb`? Or perhaps leave as is, but duplicate this information in a readme file?

Reply:
I copied the instructions for how to install this into our `Firestore/README.md`. Until there are more of these scripts (and I suspect there won't be) it seems better to keep this self-contained.